FitOn/dynamodb.py

```diff
@@ -809,6 +803,8 @@ def create_reply(thread_id, user_id, content):
     # Append the reply to the post's Replies attribute
     try:
         # Update the post by appending the new reply to the Replies list
+        # UPDATE THIS LATER
```

Comment:
This will cause an error when creating more than one reply to a post, I believe, should we update this now or later